### ISSUE TITLE
CN: example answers jump around

### ISSUE THREAD

**Issue by jdub233** [bu-ist/bu-learning-blocks#68]
The `generatePossibleAnswer() function here is producing inconsistent results:

https://github.com/bu-ist/bu-learning-blocks/blob/adb3a209ac538df791aa8d2fbbe8171e1939bc67/src/blocks/bulb-cn/CalculatedNumericAnswer.js#L11

Because the Rich Text editing components trigger the component's onChange event, every keystroke in the text entry fields causes a re-render of the component.  For answers with a non-zero "Accepted Range",  the "Example of answers..." re-renders with different content on each keystroke.

Basically the example answers jumps around with a different list as one types.

**Comment by jdub233** [comment]
If we are going to keep the "Example answers..." then `renderPossibleAnswers` needs to be re-factored to return a deterministic result.  That is, given the same inputs it must return the same result.

Right now, it is non-deterministic; it will return very different results with the same inputs.

**Comment by jdub233** [comment]
Fixed
